services/housetables/src/test/java/com/linkedin/openhouse/housetables/e2e/usertable/HtsControllerTest.java

```diff
@@ -47,7 +47,9 @@ public class HtsControllerTest {
   public void setup() {
     // TODO: Use rest API to create the table and test the find/delete user table again.
     // For now manually create the user table upfront.
-    htsRepository.save(TestHouseTableModelConstants.TEST_USER_TABLE_ROW);
+    UserTableRow testUserTableRow =
+        new TestHouseTableModelConstants.TestTuple(0).get_userTableRow();
```

Comment:
TEST_USER_TABLE_ROW is static final without any setter, is the save method using reflection to change internal elements?

Reply:
This has nothing to do with the save() method. I explicitly set the version to null in the builder and it still failed. I suspect that in this Github Action environment, all static object and its builder will always assign a default value to the inner variables, so we need to avoid abusing static objects. But anyways, let's don't spend too much time on it.